packages/keyring-eth-ledger-bridge/src/ledger-keyring.ts

```diff
@@ -448,7 +468,20 @@ export class LedgerKeyring implements Keyring {
         hdPath,
         message: remove0x(message),
       });
-    } catch (error) {
+    } catch (error: unknown) {
+      if (error instanceof TransportStatusError) {
+        const transportError: TransportStatusError = error;
+        /**
+         * for user rejected the transaction error
+         */
+        if (
+          transportError.statusCode === 27013 &&
+          transportError.message.includes('(denied by the user?) (0x6985)')
+        ) {
+          throw new Error('Ledger: User rejected the transaction');
+        }
+      }
```

Comment:
Then we can use it like:
```suggestion
      reThrowIfTransportError(error);
```